lls/nd.c

```diff
@@ -213,7 +214,7 @@ xmit_nd_req(struct gatekeeper_if *iface, const uint8_t *ip_be,
 	ipv6_hdr->payload_len = rte_cpu_to_be_16(created_pkt->data_len -
 		(l2_len + sizeof(*ipv6_hdr)));
 	ipv6_hdr->proto = IPPROTO_ICMPV6;
-	ipv6_hdr->hop_limits = IPv6_DEFAULT_HOP_LIMITS;
+	ipv6_hdr->hop_limits = lls_conf->net->ipv6_default_hop_limits;
```

Comment:
Hardcode `255` and add the comment:
`/* The IP Hop Limit field must be 255 as required by RFC 4861, sections 7.1.1 and 7.1.2. */`